lib/cron_kubernetes/cron_job.rb

```diff
@@ -5,36 +5,29 @@
 module CronKubernetes
   # A single job to run on a given schedule.
   class CronJob
-    attr_accessor :schedule, :command, :job_manifest, :name, :identifier
-
-    def initialize(schedule: nil, command: nil, job_manifest: nil, name: nil, identifier: nil)
-      @schedule     = schedule
-      @command      = command
-      @job_manifest = job_manifest
-      @name         = name
-      @identifier   = identifier
+    attr_accessor :schedule, :command, :job_manifest, :name, :identifier, :cron_job_settings
+
+    def initialize(options = {})
```

Comment:
I realized that Rubocop doesn't like lots of options on method names, but this makes the method a lot less self-documenting.